client/src/components/Tickets/TicketModal.js

```diff
@@ -147,7 +162,10 @@ const TicketHistoryCard = ({
   const ticketHeaderClassName = ticketCreator
     ? 'ticket-creator'
     : 'ticket-responder';
-
+  const filteredattachment = (attachments || []).filter(
```

Comment:
Can you either add a comment that explains what you're filtering here, or rename this variable to be more descriptive of what's being filtered? For example, `attachmentTitles`